samples/communication-between-nodes-1/src/main/java/com/badoo/ribs/samples/comms_nodes_1/rib/container/ContainerBuilder.kt

```diff
@@ -0,0 +1,47 @@
+@file:SuppressWarnings("LongParameterList")
+
+package com.badoo.ribs.samples.comms_nodes_1.rib.container
+
+import com.badoo.ribs.builder.SimpleBuilder
+import com.badoo.ribs.core.modality.BuildParams
+import com.badoo.ribs.routing.source.backstack.BackStack
+import com.badoo.ribs.samples.comms_nodes_1.rib.container.routing.ContainerChildBuilders
+import com.badoo.ribs.samples.comms_nodes_1.rib.container.routing.ContainerRouter
+import com.badoo.ribs.samples.comms_nodes_1.rib.container.routing.ContainerRouter.Configuration
+import com.badoo.ribs.samples.comms_nodes_1.rib.container.routing.ContainerRouter.Configuration.Content
+
+class ContainerBuilder(
+    private val dependency: Container.Dependency
+) : SimpleBuilder<Container>() {
+
+    private val builders by lazy { ContainerChildBuilders(dependency) }
+
+    override fun build(buildParams: BuildParams<Nothing?>): Container {
+
+        val backStack = backStack(buildParams)
+        val router = router(buildParams, backStack, builders)
+        val presenter = ContainerPresenterImpl(backStack)
+
+        return ContainerNode(
+            buildParams = BuildParams.Empty(),
```

Comment:
Never use BuildParams.Empty()! (Check its description). Pass the instance you get in this method.
```suggestion
            buildParams = buildParams,
```

Reply:
Oh, I totally missed the description. Thanks for this!
Maybe it would be a good idea to have the `.Empty()` builder just in the test package? or `@Deprecate` it so you have a visual indicator in the IDE about that method?